docs/sources/configure-server/about-server-api.md

```diff
@@ -0,0 +1,194 @@
+---
+description: Learn about the Pyrocope server API
+menuTitle: Server API Overview
+title: Pyroscope Server HTTP API Reference
+weight: 20
+---
+
+# Pyroscope Server HTTP API Reference
+
+Pyroscope server exposes an HTTP API that can be used to query profiling data and to ingest profiling data from other sources.
+
+## Authentication
+TODO - how does authentication work with new Grafana cloud stuff?
```

Comment:
```suggestion
Grafana Pyroscope does not come with any included authentication layer. Operators are expected to run an authenticating reverse proxy in front of your services.

Note that when using Pyroscope in multi-tenant mode, Pyroscope requires the HTTP header `X-Scope-OrgID` to be set to a string identifying the tenant; the responsibility of populating this value should be handled by the authenticating reverse proxy. Please refer to the [multi-tenancy documentation](https://grafana.com/docs/pyroscope/latest/configure-server/about-tenant-ids) for more information.
```

I'm not sure we should cover Grafana Cloud auth here. Should we maybe update the hosted instance page (where the credentials are obtained) with some examples?